rust/sealevel/libraries/hyperlane-sealevel-token/src/processor.rs

```diff
@@ -225,38 +245,33 @@ where
             mailbox_message_dispatch_authority_pda_seeds!(dispatch_authority_bump),
         )?;
 
-        let token: HyperlaneToken<T> = HyperlaneToken {
-            bump: token_bump,
-            mailbox: init.mailbox,
-            mailbox_process_authority,
-            dispatch_authority_bump,
-            owner: Some(*payer_account.key),
-            interchain_security_module: init.interchain_security_module,
-            decimals: init.decimals,
-            remote_decimals: init.remote_decimals,
-            remote_routers: HashMap::new(),
-            plugin_data,
-        };
-        HyperlaneTokenAccount::<T>::from(token).store(token_account, true)?;
+        token_account_data.store_with_rent_exempt_realloc(token_account, &rent, payer_account)?;
```

Comment:
Reallocing shouldn't be required here since it's the initialization step, but I assume it's used for consistency with the other instructions?

Reply:
Ah good point, i'll remove, no need for this